### PR TITLE
Features/new stats app

### PR DESCRIPTION
**Pull request by baktun14**
- Added new stats app
- https://stats.akash.network

Main features:
- Migration from next <13 to next 14 (pages to app folder)
- Migration from Material-ui to TailwindCSS + shadcn
- Integrate shadcn components

### CODE REVIEW

File: stats-web/src/app/addresses/[address]/transactions/AddressTransactions.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                <TableHead className="text-center">Result</TableHead>
+                <TableHead className="text-center">Amount</TableHead>
+                <TableHead className="text-center">Fee</TableHead>
+                <TableHead className="text-center">Height</TableHead>
+                <TableHead className="text-center">Time</TableHead>
+              </TableRow>
+            </TableHeader>
+
+            <TableBody>{transactionsResult?.results.map(tx => <TransactionRow key={tx.hash} transaction={tx} blockHeight={tx.height} />)}</TableBody>
+            {/* <TableBody>{deploymentsResult?.results.map(deployment => <DeploymentRow key={deployment.dseq} deployment={deployment} />)}</TableBody> */}
```
Comment (baktun14) [inline stats-web/src/app/addresses/[address]/transactions/AddressTransactions.tsx]: ✂️

File: stats-web/src/app/addresses/[address]/transactions/AddressTransactions.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        {isLoading && (
+          <div className="flex items-center justify-center p-4">
+            <Spinner size="large" />
+          </div>
+        )}
+        {/* {!!pageCount && (
+          <div className={classes.pagerContainer}>
+            <Pagination count={pageCount} page={page} onChange={handlePageChange} size="medium" />
+          </div>
+        )} */}
```
Comment (baktun14) [inline stats-web/src/app/addresses/[address]/transactions/AddressTransactions.tsx]: :scissors:

File: stats-web/src/components/table/data-table-row-actions.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import { DotsHorizontalIcon } from "@radix-ui/react-icons";
+import { Row } from "@tanstack/react-table";
+// import { labels } from "../data/data";
+// import { taskSchema } from "../data/schema";
```
Comment (baktun14) [inline stats-web/src/components/table/data-table-row-actions.tsx]: ✂️

File: stats-web/src/lib/apiUtils.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  }
+  static mainnetVersion() {
+    return `${BASE_API_URL}/getMainnetVersion`;
+  }
+  static testnetVersion() {
+    return `${BASE_API_URL}/getTestnetVersion`;
+  }
+  static sandboxVersion() {
+    return `${BASE_API_URL}/getSandboxVersion`;
+  }
```
Comment (baktun14) [inline stats-web/src/lib/apiUtils.ts]: ✂️ ?

File: stats-web/src/store/networkStore.ts [hunk 5]
```diff
@@ -0,0 +1,66 @@
+// import { Network } from "@src/types/network";
+// import { ApiUrlService } from "@src/utils/apiUtils";
```
Comment (baktun14) [inline stats-web/src/store/networkStore.ts]: ✂️

File: stats-web/src/app/addresses/[address]/transactions/AddressTransactions.tsx (hunk 1 above)
Comment (baktun14) [reply stats-web/src/app/addresses/[address]/transactions/AddressTransactions.tsx]: Must implement custom datatable component for address transactions

File: stats-web/src/app/addresses/[address]/deployments/columns.tsx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+import Link from "next/link";
+import { z } from "zod";
+import { statuses } from "./data";
+import { Badge } from "@/components/ui/badge";
+import { LeaseSpecDetail } from "@/components/LeaseSpecDetail";
+import { roundDecimal } from "@/lib/mathHelpers";
+import { bytesToShrink } from "@/lib/unitUtils";
+import { cn } from "@/lib/utils";
+
+export const transactionRowSchema = z.object({
```
Comment (Redm4x) [inline stats-web/src/app/addresses/[address]/deployments/columns.tsx]: Should it be `deploymentRowSchema`?

File: stats-web/src/app/transactions/page.tsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import PageContainer from "@/components/PageContainer";
+import { Title } from "@/components/Title";
+import { Metadata } from "next";
+import { TransactionsTable } from "./TransactionsTable";
+
+type Props = {};
+
+export const metadata: Metadata = {
+  title: "Transasctions"
```
Comment (Redm4x) [inline stats-web/src/app/transactions/page.tsx]:
```suggestion
  title: "Transactions"
```

**Review by Redm4x — APPROVED** [review]

**Pull request merged**